pkg/log/context.go

```diff
@@ -36,3 +58,40 @@ func goid() string {
 	idField := strings.Fields(strings.TrimPrefix(string(buf[:n]), "goroutine "))[0]
 	return idField
 }
+
+func paramKey(name string) string {
+	return PARAM + name
+}
+
+func GetParamFromContext(ctx context.Context, key string) (string, string) {
+	val := ctx.Value(contextKey(paramKey(key)))
+	if val == nil {
+		return key, ""
+	} else {
+		return key, val.(string)
+	}
+}
+
+//openapi middleware handler that takes URL params from the path and put them
```

Comment:
Comment is wrong.
This is taking specific params from the matched-route and placing in the context